build/rust/rust_cxx.gni

```diff
@@ -0,0 +1,109 @@
+# Copyright 2021 The Chromium Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+import("//brave/build/cargo.gni")
+
+cxx_build_path = "$root_build_dir/cxxbridge"
+cxxbridge_executable = "$cxx_build_path/$rustc_target/release/cxxbridge"
```

Comment:
If you build a debug build this path is actually `"$cxx_build_path/$rustc_target/debug/cxxbridge"` Not sure if we want to ensure we always build release for the rust part, or adjust the path based on build type. cc @bridiver

Reply:
this should be getting pulled from cargo now; will mark as resolved